.gitpod.yml

```diff
@@ -0,0 +1,6 @@
+vscode:
+  extensions:
+    - matklad.rust-analyzer
+    - serayuzgur.crates
+    - Swellaby.vscode-rust-test-adapter
+    - hbenl.vscode-test-explorer
```

Comment:
What do these extensions do?

Reply:
They install the necesary tools into VScode:
    - serayuzgur.crates:  Autocomplete cargo crates in the toml
    - Swellaby.vscode-rust-test-adapter: Run tests from the gui.
    - hbenl.vscode-test-explorer: test report in the gui too.